lambdas/evaluate-gpg45-scores/src/main/java/uk/gov/di/ipv/core/evaluategpg45scores/EvaluateGpg45ScoresHandler.java

```diff
@@ -232,7 +234,7 @@ private JourneyResponse checkForMatchingGpg45Profile(
                 gpg45ProfileEvaluator.getFirstMatchingProfile(
                         gpg45Scores, CURRENT_ACCEPTED_GPG45_PROFILES);
 
-        if (matchedProfile.isPresent()) {
+        if (matchedProfile.isPresent() && !checkRequiresAdditionalEvidence(userId)) {
```

Comment:
I thought moving this check after the matched profile would be more efficient in terms of performance. Because, currently, it should only be true for the BAV CRI? But, if I'm mistaken, I can pull check forward before matching the profile.